### ISSUE TITLE
Inviting to GDrive based on google group `contributors@crypto-kaizen.com`

### ISSUE THREAD

**Issue by samarth9008** [sorrentum/sorrentum#508]

**Comment by samarth9008** [comment]
I don't think we required this issue because we will be using google group to share permission for the google drive which can be done manually since there is just one group. All the people in that group will automatically get the desired permission. wdyt @DanilYachmenev ??

**Comment by DanilYachmenev** [comment]
Hmm its hard o say until we check how this functionality works and what should be the process
Could you describe here pls, how i your opinion the process would look like in steps?

Then, lets think which steps could be automated, which ones would require a separate simple script and which can be just a one-liner in the master script

**Comment by samarth9008** [comment]
We want to give active contributors access to this gDrive. The active contributors will be in the google group. So instead of giving access to single users, we should just give access to the mail of the group. In that way only the people inside the group will be given access. As soon as they are removed from the group there permissions will be automatically revoked from the drive.

**Comment by samarth9008** [comment]
Also we are creating a script which will remove people from that group.
